static/app/views/performance/transactionSummary/utils.tsx

```diff
@@ -149,15 +149,23 @@ export function generateReplayLink(routes: PlainRoute<any>[]) {
     const replaySlug = `${tableRow['project.name']}:${replayId}`;
     const referrer = encodeURIComponent(getRouteStringFromRoutes(routes));
 
+    const transactionTimestamp = tableRow.timestamp
+      ? new Date(tableRow.timestamp).getTime()
+      : 0;
+
+    // Substract duration from timestamp to get the start time of the transaction
+    const transactionStartTimestamp =
+      transactionTimestamp - (tableRow['transaction.duration'] as number);
```

Comment:
The comment isn't really needed. We've got well-named variables already!

Also, should we care about the case where `transactionTimestamp` could be `0`, and we're subtracting from that? the result will be a negative number.

Reply:
Added new condition to prevent check on this